addons/sys_core/stringtable.xml

```diff
@@ -113,124 +113,28 @@
-            <Russian>Альт. радио кнопка 1</Russian>
-            <Chinese>設置備用無線電按鍵1</Chinese>
-            <Chinesesimp>设置备用无线电按键1</Chinesesimp>
-            <Portuguese>Tecla do  Rádio Alternativa 1</Portuguese>
+            <English>Radio 1 PTT</English>
         </Key>
         <Key ID="STR_ACRE_sys_core_AltPTTKey2">
-            <English>Alt Radio Key 2</English>
-            <Spanish>Tecla de transmisión alternativa 2</Spanish>
-            <Czech>Alternativní klávesa rádia 2</Czech>
-            <German>Alternative Sprechtaste 2</German>
-            <Japanese>代替無線キー 2</Japanese>
-            <Italian>Trasmetti sulla Radio alternativa n.2</Italian>
-            <French>Touche radio alternative 2</French>
-            <Korean>대체 무전 단축키 2</Korean>
-            <Polish>Dodatkowy klawisz radia 2</Polish>
-            <Russian>Альт. радио кнопка 2</Russian>
-            <Chinese>設置備用無線電按鍵2</Chinese>
-            <Chinesesimp>设置备用无线电按键2</Chinesesimp>
-            <Portuguese>Tecla do  Rádio Alternativa 2</Portuguese>
+            <English>Radio 2 PTT</English>
         </Key>
         <Key ID="STR_ACRE_sys_core_AltPTTKey3">
-            <English>Alt Radio Key 3</English>
-            <Spanish>Tecla de transmisión alternativa 3</Spanish>
-            <Czech>Alternativní klávesa rádia 3</Czech>
-            <German>Alternative Sprechtaste 3</German>
-            <Japanese>代替無線キー 3</Japanese>
-            <Italian>Trasmetti sulla Radio alternativa n.3</Italian>
-            <French>Touche radio alternative 3</French>
-            <Korean>대체 무전 단축키 3</Korean>
-            <Polish>Dodatkowy klawisz radia 3</Polish>
-            <Russian>Альт. радио кнопка 3</Russian>
-            <Chinese>設置備用無線電按鍵3</Chinese>
-            <Chinesesimp>设置备用无线电按键3</Chinesesimp>
-            <Portuguese>Tecla do  Rádio Alternativa 3</Portuguese>
+            <English>Radio 3 PTT</English>
         </Key>
         <Key ID="STR_ACRE_sys_core_AltPTTKey1_description">
-            <English>Use the first radio in your inventory to talk.</English>
-            <Spanish>Utiliza la primera radio en el inventario para hablar</Spanish>
-            <Czech>Použít první rádio v inventáři k mluvení.</Czech>
-            <German>Das erste Funkgerät zum Sprechen verwenden.</German>
-            <Japanese>インベントリの1番目にある無線機を使って話します。</Japanese>
-            <Italian>Trasmetti sulla Prima Radio nel tuo equipaggiamento.</Italian>
-            <French>Utiliser la première radio dans l'inventaire pour parler.</French>
-            <Korean>인벤토리의 첫 번째에 위치한 무전기를 사용해 대화하세요.</Korean>
-            <Polish>Użyj aby nadawać na pierwszym radiu z Twojego ekwipunku.</Polish>
-            <Russian>Использовать первое радио из инвентаря.</Russian>
-            <Chinese>使用物品欄內第一個無線電進行通話。</Chinese>
-            <Chinesesimp>使用物品栏内第一个无线电进行通话。</Chinesesimp>
-            <Portuguese>Usar o primeiro rádio do inventário para falar.</Portuguese>
+            <English>Activate first radio push-to-talk.</English>
```

Comment:
this should perhaps mention that it also "selects" the radio if that's still the case?

Reply:
Yes, functionality of keybinds itself didn't change.